rust/core-lib/src/config.rs

```diff
@@ -308,12 +308,14 @@ impl ConfigManager {
     /// Sets a specific language for the given buffer. This is used if the
     /// user selects a specific language in the frontend, for instance.
     #[allow(dead_code)]
```

Comment:
we can remove this `#[allow(..)]` now; it was in place because I wrote this function at the same time as a bunch of other config stuff, and thought it was useful to keep it in place even though we weren't using it; now that we're using this flag (that silences a compiler warning) is unnecessary.